src/templates/admin/elements/publish/pubdate.html

```diff
@@ -18,21 +17,8 @@ <h5>Publication Date/Time</h5>
             {% endif %}
             <form method="POST">
                 {% csrf_token %}
-                <div class="callout">
-                    <div class="row">
-                        <div class="large-8 columns">
-                            <label for="date">Date</label>
-                            <input type="text" name="date" id="date" required {% if article.date_published %}value="{{ article.date_published|date:'Y-m-d' }}"{% endif %}>
-                        </div>
-                        <div class="large-4 columns">
-                            <label for="time">Time</label>
-                            <input type="text" name="time" id="time" required {% if article.date_published %}value="{{ article.date_published|date:'H:i' }}"{% endif %}>
-                        </div>
-                        <div class="large-12 columns">
-                            <button type="submit" name="pubdate" class="small success button">Set Publication Date</button>
-                        </div>
-                    </div>
-                </div>
+                {{ pub_date_form }}
```

Comment:
This is being output in table format. Perhaps use as_p or the foundation tag?
<img width="1110" alt="Screenshot 2024-05-29 at 13 26 41" src="https://github.com/BirkbeckCTP/janeway/assets/8321378/eb086602-4eb6-478f-a60a-e0bfcbe5dc1a">

Reply:
Huh, that is wild. Firefox clears it in my inspector pane (after parsing the HTML) but I can see it when I inspect the source. Is this a Django default feature, or something we have configured? In any case, I've added the foundation filter.